roles/zookeeper-exporter/tasks/main.yml

```diff
@@ -0,0 +1,33 @@
+- include_role: 
+    name: gantsign.golang
+
+- name: ensure Prometheus exporter directory exist
+  file:
+    path: "{{ zookeeper_exporter_install_dir }}"
+    state: directory
+
+- name: Get the zookeeper package
+  shell: "{{ golang_install_dir }}/bin/go get -u {{ zookeeper_exporter_url }}"
+  args:
+    executable: /bin/bash
+
+- name: Move zookeeper build package to exporter directory
```

Comment:
do you not need to build the package? https://github.com/cloudflare/kafka_zookeeper_exporter#building

Reply:
`go get -u` take cares of building it while downloading.